developers/integrations/llm-frameworks/semantic-kernel/semantic-kernel.md

```diff
@@ -0,0 +1,31 @@
+---
+title: Semantic Kernel
+sidebar_position: 6
+---
+Semantic Kernel is an LLM framework developed by Microsoft. Semantic Kernel facilitates building LLM applications with abstractions such as `plugins`, `memory`, `planners`, and more! 
+
+## Semantic Kernel and Weaviate
+Weaviate is a supported vector store in Semantic Kernel. 
+
+
+## Our Resources 
+The resources are broken into two categories: 
+1. [**Hands on Learning**](#hands-on-learning): Build your technical understanding with end-to-end tutorials.
+
+2. [**Read and Listen**](#read-and-listen): Develop your conceptual understanding of these technologies.
+
+### Hands on Learning
+
+| Topic | Description | Resource | 
+| --- | --- | --- |
+|  RAG Chatbot with Semantic Kernel | Implements a simple workflow of retrieve then generate. using semantic kernel to help with prompt engineering and orchestrating OpenAI API calls and Weaviate as knowledgebase from which to retreive semantically relevant context. | [Notebook](https://github.com/weaviate/recipes/blob/main/integrations/llm-frameworks/semantic-kernel/dotnet/Chatbot_RAG_Weaviate.ipynb) |
```

Comment:
Typo and consider
retrieve

Implements a simple workflow of retrieve-then-generate. Use a semantic kernel for prompt engineering, orchestrating OpenAI API calls, and integrating Weaviate. Use Weavate as a knowledge base to retrieve semantically relevant context.